noteDB.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE IF NOT EXISTS 'notesDB';
```

Comment:
Not sure if its good to put the database name in single quotes.

Reply:
Me neither, this is the syntax I found online. I'll look it up

Reply:
According to w3schools you don't even need the single quotes.
https://www.w3schools.com/sql/sql_create_db.asp

```sql
CREATE DATABASE databasename; 
```